sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
```

Comment:
Oh I'm silly! So x/sys/windows GetTokenUser is already there. When in Rome I guess :) 

The `go x/foo/bar` thing threw me, definitely new to go.